map_gen/maps/crash_site/outpost_builder.lua

```diff
@@ -1445,14 +1479,21 @@ Public.magic_item_crafting_callback =
         entity.operable = false
 
         local recipe = callback_data.recipe
-        if recipe then
+        local has_fluid_output = callback_data.keep_active
+        if recipe and not has_fluid_output then -- to avoid trying to put an item into a fluid inventory
             entity.set_recipe(recipe)
-        else
-            local furance_item = callback_data.furance_item
-            if furance_item then
+            local output_inv = entity.get_output_inventory()
+            output_inv.insert {name = callback_data.output.item, count = 200}
```

Comment:
If you just want a full stack you can just pass in a string.
```suggestion
            output_inv.insert(callback_data.output.item)
```